napari/_vispy/vispy_welcome_visual.py

```diff
@@ -0,0 +1,115 @@
+from os.path import dirname, join
```

Comment:
Is this file meant to be here ?

Reply:
I think this is our old welcome visual which isn't in use anymore - we can probably delete

Reply:
Ahhh, I put it back in on the rebase. My bad!

Reply:
Fixed